### PR TITLE
add Kokoro config for OpenJDK11

### PR DESCRIPTION
**Pull request by elharo**
@acozzette Please consider whether this is the correct Dockerfile for this job. For JDK17 (coming up soon) I will need to either modify this one or add a new one that includes OpenJDK 17. 

Also, I'm assuming we put this in Github first and then add the configs in piper.

### CODE REVIEW

File: kokoro/linux/java_jdk11/continuous.cfg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Config file for running tests in Kokoro
+
+# Location of the build script in repository
+build_file: "protobuf/kokoro/linux/java_linkage_monitor/build.sh"
```
Comment (acozzette) [inline kokoro/linux/java_jdk11/continuous.cfg:4]: I think this should be: `protobuf/kokoro/linux/java_jdk11/build.sh`

File: kokoro/linux/java_jdk11/presubmit.cfg [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Config file for running Linkage Monitor in Kokoro
+# https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/linkage-monitor
+
+# Location of the build script in repository
```
Comment (acozzette) [inline kokoro/linux/java_jdk11/presubmit.cfg:4]: This path needs an update, too.

**Review by acozzette — APPROVED** [review]

File: kokoro/linux/java_jdk11/continuous.cfg (hunk 1 above)
Comment (elharo) [reply kokoro/linux/java_jdk11/continuous.cfg:4]: done

File: kokoro/linux/java_jdk11/presubmit.cfg (hunk 2 above)
Comment (elharo) [reply kokoro/linux/java_jdk11/presubmit.cfg:4]: done

**Pull request merged**